project/test_framework/venv/pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = E:\Program Files\Python\Python36-32
```

Comment:
I would prefer not to keep the venv files under revision control. Rather, we should see if we can keep a virtual environment description file, such as a [conda](https://conda.io/docs/) `environment.yml` or a [pipenv](https://github.com/pypa/pipenv) file. That way, all dependencies are downloaded from external source, rather than kept in our repository.